sdk/scheduler/src/main/java/com/mesosphere/sdk/api/OfferOutcomeResource.java

```diff
@@ -0,0 +1,34 @@
+package com.mesosphere.sdk.api;
+
+import com.mesosphere.sdk.offer.history.OfferOutcomeTracker;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/v1/debug/offers")
+public class OfferOutcomeResource {
+    private static final Logger LOGGER = LoggerFactory.getLogger(OfferOutcomeResource.class);
+
+    private final OfferOutcomeTracker offerOutcomeTracker;
+
+    public OfferOutcomeResource(OfferOutcomeTracker offerOutcomeTracker) {
+        this.offerOutcomeTracker = offerOutcomeTracker;
+    }
+
+    /**
+     * Renders the current set of offer outcomes as an HTML <table>.
+     * @return HTML response of the table.
+     */
+    @GET
+    public Response getOfferOutcomes(@QueryParam("json") boolean json) {
+        if (json) {
+            return ResponseUtils.jsonOkResponse(offerOutcomeTracker.toJson());
+        } else {
+            return ResponseUtils.htmlOkResponse(offerOutcomeTracker.toHtml());
```

Comment:
I may have missed the discussion as to why the JSON output is not sufficient?

Reply:
Support + others have indicated that having a human consumable (e.g. HTML) endpoint is useful to them. I know that some users wont have the CLI Installed, so there's some utility in having it available directly from the service endpoint. If the CLI were to dump this out, it will obviously just fetch and parse the JSON.

Reply:
👍